### DISCUSSION TITLE
Comments/Notes or Custom Tags on Media

### DISCUSSION THREAD

**Discussion by FlamingBluejay** [seerr-team/seerr#758] [Ideas]
I think an interesting idea would be to implement the ability to make comments/notes to media within Jellyseerr. This could work by having a section under the media (maybe above or below the tags?) where each user can make comments/notes for themselves. For example, sometimes I have media that runs into problems when I have tried to download it because maybe it doesn't have enough seeders or I can't find a file size that I want. I could use this to remind myself which episodes/seasons I have problems with for a certain show or what the problem is for a movie. I could see this also being useful if I wanted to remind myself that I wanted to download this particular show/movie in a different format or just whether or not I really liked a particular show/movie or not. 

Alternatively, the ability to add custom tags to media could accomplish this same goal and might look a little cleaner. For example, imagine that you really like a certain show and you want to remember that you liked this show to find similar content in the future. Then you could potentially add a "Loved This" tag to that show. Or maybe you watched one episode of a different show and hated it. Maybe you could add a custom tag that says "Hate This" where it applies that custom tag so you know to avoid downloading it again in the future.

The watchlist accomplishes a similar goal to this but it's limited in that I only really use it for media that I plan to download/watch in the future and not necessarily media I have already watched. I know that it is possible to edit the tags in Jellyfin by editing the metadata, but I was imagining a more streamlined implementation that doesn't change the source tags. Additionally, some way to distinguish that these tags are "custom" and not baked in from the media source might also be useful since maybe you could filter a search to find media with particular tags or all media with custom tags.

**Comment by fallenbagel** [comment]
About the comments/notes when an issue occurs, we already have a similar feature. Report button. You can raise an issue and you can comment there and resolve

**Reply by FlamingBluejay** [reply]
You're completely right! I haven't used that feature before and I didn't realize it does this. I still think that the custom tags could be cool, also.
